### PR TITLE
[Bugfix] Bilinear resize bug fix from PR #2777

### PR DESCRIPTION
**Pull request by Laurawly**
@FrozenGene Please review.

### CODE REVIEW

**Comment by FrozenGene** [comment]
Thanks @Laurawly generally looks good to me and could solve the issues mentioned. However, could we add one unit testing of resize bilinear to compare MXNet forward? I think it will be more nice.

**Comment by Laurawly** [comment]
@FrozenGene Does the test look good to you or more tests are needed?

File: tests/python/frontend/mxnet/test_forward.py [hunk 1]
```diff
@@ -464,6 +464,12 @@ def verify(data_shape, weight_shape):
     verify((2, 2), (4, 5))
     verify((2, 3, 4), (4, 5))
 
+def test_forward_bilinear_resize():
```
Comment (FrozenGene) [inline tests/python/frontend/mxnet/test_forward.py]: I think we can provide one new more unit test with `scale_height` and `scale_width` to cover our implementation.

**Comment by Laurawly** [comment]
@FrozenGene Mxnet gives me no "scale_height/width" parameter error locally. I think maybe it depends on the version of Mxnet.

File: tests/python/frontend/mxnet/test_forward.py (hunk 1 above)
Comment (Laurawly) [reply tests/python/frontend/mxnet/test_forward.py]: Does CI has the latest mxnet installed?

Comment (Laurawly) [reply tests/python/frontend/mxnet/test_forward.py]: I'll just try and see if it gives me error.

Comment (Laurawly) [reply tests/python/frontend/mxnet/test_forward.py]: @FrozenGene seems https://mxnet.apache.org/api/python/symbol/contrib.html?highlight=bilinearresize#mxnet.symbol.contrib.BilinearResize2D is not giving the correct guidance, it's not related to mxnet version.

**Comment by FrozenGene** [comment]
@Laurawly I use MXNet 1.4 and also get the same error. Seems that it is MXNet's doc issue. I think we test `height` `width` is enough currently.

File: tests/python/frontend/mxnet/test_forward.py (hunk 1 above)
Comment (Laurawly) [reply tests/python/frontend/mxnet/test_forward.py]: @FrozenGene So I confirmed that only Mxnet 1.5 fixes the bugs. If CI can update mxnet, we can then have this test.

**Comment by Laurawly** [comment]
> @Laurawly I use MXNet 1.4 and also get the same error. Seems that it is MXNet's doc issue. I think we test `height` `width` is enough currently. However, could we construct another testing? i.e. currently , we are `data -> resize bilinear` , `data` is variable. We could add `data -> add -> resize bilinear`. i.e. `resize bilinear` accepts one op's output. Then we could cover all conditions of input.

I confirmed with mxnet developers and confirmed that it's a bug. Mxnet 1.5 solves this issue. If we could upgrade ci to use mxnet 1.5 (pre-release) problem will be solved.

**Comment by Laurawly** [comment]
@FrozenGene @tqchen Is it possible to update the CI to use mxnet 1.5? Or I remove the test for now?

**Comment by FrozenGene** [comment]
I personally suggest remove the test of `scale_height / scale_width`. Because current stable version of MXNet doesn’t have it. Also want to hear @tqchen’s comment.

**Comment by tqchen** [comment]
Let us remove the test

**Comment by Laurawly** [comment]
@FrozenGene Test removed.

**Comment by yzhliu** [comment]
Please kick the CI again.

File: tests/python/frontend/mxnet/test_forward.py [hunk 2]
```diff
@@ -506,4 +513,8 @@ def test_forward_smooth_l1():
     test_forward_broadcast_axis()
     test_forward_full()
     test_forward_embedding()
+<<<<<<< a07df95a837dceb70d8f36c33fe680a913faf2b9
```
Comment (icemelon9) [inline tests/python/frontend/mxnet/test_forward.py]: Need to remove this line.

Comment (Laurawly) [reply tests/python/frontend/mxnet/test_forward.py]: Thanks for reminding!

File: tests/python/frontend/mxnet/test_forward.py (hunk 1 above)
Comment (icemelon9) [reply tests/python/frontend/mxnet/test_forward.py]: Could you add this in the comment of this issue?

File: tests/python/frontend/mxnet/test_forward.py [hunk 3]
```diff
@@ -505,6 +505,12 @@ def verify(xshape, yshape, y_data):
     verify((2, 2), (2, 3), [[1, 1, 0], [0, 1, 0]])
     verify((2, 2, 2), (2, 2), [[0, 1], [1, 0]])
 
+def test_forward_bilinear_resize():
+    # add tests including scale_height and scale_width when mxnet is updated to version 1.5
```
Comment (Laurawly) [inline tests/python/frontend/mxnet/test_forward.py]: @icemelon9 Please check if this looks ok.

Comment (icemelon9) [reply tests/python/frontend/mxnet/test_forward.py]: Already approved. Let's wait for CI to complete.

**Comment by icemelon9** [comment]
Thanks everyone. this is now merged

**Pull request merged**